tests/suites/1_stateful/00_copy/00_0000_copy_from_ipfs_location.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+. "$CURDIR"/../../../shell_env.sh
+TABLE=ontime200
+FILE=ontime_200
+QMHASH=Qmei4dyyPazUy24cfCAtmumC1Ff1VLLiqjzF1zCXXVtvSk
```

Comment:
Where is this hash from?

Reply:
I'm guessing this hash is not reliable.

Please use this https://ipfs.filebase.io/ipfs/QmPpCt1aYGb9JWJRmXRUnmJtVgeFFTJGzWFYEEX7bo9zGJ for test.

Reply:
100000 lines of data are too large and will make the CI slow, so I uploaded the testing files on my computer. But...well seems like I know nothing about IPFS...